backend/src/main/java/com/filegenie/backend/Controllers/RegisterFieldController.java

```diff
@@ -0,0 +1,66 @@
+package com.filegenie.backend.Controllers;
+
+import com.filegenie.backend.DTO.HttpException;
+import com.filegenie.backend.Entities.Field;
+import com.filegenie.backend.Entities.User;
+import com.filegenie.backend.Services.JsonParserService;
+import com.filegenie.backend.Services.RegisterFieldService;
+import com.filegenie.backend.Services.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.util.List;
+
+@Controller
+@RestController
+@RequestMapping("/api/register")
+public class RegisterFieldController {
```

Comment:
```suggestion
@RequestMapping("/api/file/import")
public class ImportFileController {
```

(faudrait aussi renommer le fichier `.java` dans ce cas)